packages/components/popper/src/content.vue

```diff
@@ -96,6 +104,8 @@ export default defineComponent({
             })
             popperInstanceRef.value = newInstance
 
+            watch(() => props.visible, togglePopoerAlive, { immediate: true })
+
```

Comment:
Why are we using nested watch? I don't think this is a good practice, can we move it outside this watch call? We can watch it independently after this watch.